src/bin/mrtg

```diff
@@ -1666,10 +1664,16 @@ TEXT
     $$rcfg{'rgb5'}{$router} = "" unless defined $$rcfg{'rgb5'}{$router};
     $$rcfg{'rgb6'}{$router} = "" unless defined $$rcfg{'rgb6'}{$router};
 
-    print HTML "
-		<style type=\"text/css\">
+    print HTML "		<style>
 			body {
 				background-color: $$rcfg{'backgc'}{$router};
+    ";
+    if($$rcfg{'backgc'}{$router} eq '#ffffff') {
```

Comment:
This comparison doesn't work very well for light backgrounds that are not white. I use an off-white background, and this change has resulted in the text being unreadable.

Reply:
if you like to suggest something more generic ... PRs are always welcome

Reply:
There is a typical CSS problem in previous version of mrtg: defined background-color but not defined text color. It forces use always white background regardless of the user's needs. Webdesigner should define both colors with sufficient contrast, or none of them to use the default browser color setting. Version 2.17.7 has a simple workaround of mrtg.cfg missing text color property with assigned background color (backgc) property.

We have three solutions:
1. Ensure sufficient contrast for all background color values by better formula (lines 1671 - 1675) 
2. Add mrtg.cfg text color property + ensure sufficient contrast for empty text color value (solution 1).
3. Remove both colors, set empty values and ignore mrtg.cfg backgc property (lines 1658 - 1675)

Which solution do you propose? I can do changes to source code according to consensus.

Reply:
I think adding an optional  configuration property for the foreground color would be the least intrusive ...

Reply:
sgtm, if you want an automatic switch like this then you need to look at if it's "more than 50% brightness" to choose the contrasting color, but that requires parsing the color and converting RGB to HSB.

Reply:
Exactly... already done... waiting for pull-request to be merged.